client/app/dedicated/server/bandwidth/cancel-private/dedicated-server-bandwidth-vrack-cancel.controller.js

```diff
@@ -25,19 +27,16 @@ class ServerCancelBandwidthVrackCtrl {
       },
     ];
 
-    // Ugly patch... the wizard won't resolve its step-on-load event handler
-    // if it isn't on the scope..
-    this.$scope.initUser = this.steps[0].load.bind(this);
-
-    // Same thing for reset action and open openBC
-    this.$scope.cancelOption = this.cancelOption.bind(this);
+    this.$scope.initUser = () => this.steps[0].load();
```

Comment:
IMO, Use of `$scope` in a component is not a good idea

Reply:
I'll refactor a bit those parts